### ISSUE TITLE
Add note about window.location.*

### ISSUE THREAD

**Issue by johnbintz** [johnbintz/jasmine-headless-webkit#63]
Since you're on a file, there is no URI, and one should mock/stub, or use PhantomJS.

**Comment by lmarburger** [comment]
How might one stub/mock window.location.*? I'm playing with this now and it seems most things try to do `window.location = function() {...}` which makes webkit think it's leaving the page.

**Comment by johnbintz** [comment]
What library/code are you using that is working with `window.location`? I'd like to take a look.

**Comment by lmarburger** [comment]
Oh sorry I wasn't clear. I tried to use `spyOn` and that obviously failed. I just need to read the value of `window.location.search` so I broke the first rule of the Mockist and moved it into a private method mocking its response.

**Comment by johnbintz** [comment]
Yeah, I usually end up taking one for the team when it comes to `window.location` anyway and wrap up any uses of it in some other function, then test for calls to that function. I live dangerously.
